modules/social_features/social_comment_upload/src/SocialCommentUploadConfigOverride.php

```diff
@@ -41,15 +41,11 @@ public function __construct(ConfigFactoryInterface $config_factory) {
   public function loadOverrides($names) {
     $overrides = [];
 
-    // We don't need to add any fields if uploads are disabled.
-    // We use getEditable and getOriginal to avoid override loops.
-    if ($this->configFactory->getEditable('social_comment_upload.settings')->getOriginal('allow_upload_comments', TRUE) == FALSE) {
-      return $overrides;
-    }
-
     // Add field_group and field_comment_files.
+    // We don't need to add any fields if uploads are disabled, so let's check.
```

Comment:
Bram, I think that this comment should be "We need to add" right? I see that the if logic changed from == FALSE to == TRUE